content/installation/setup/installation/UsingZooKeeperProperties.md

```diff
@@ -0,0 +1,55 @@
+<!--
+title: "Configuring Contrast To Use ZooKeeper"
+description: "Instructions for configuring TeamServer to use ZooKeeper as the single source of truth"
+tags: "installation EOP distributed configuration ZooKeeper"
+-->
+
+## About ZooKeeper and Contrast
+This guide is for enterprise-on-premises (EOP) administrators who are looking to move to a distributed setup of Contrast and want to use ZooKeeper as their configuration management. 
+
+> **Note:** While the following steps guide you through the setup and configuration of additional software, keep in mind that you're responsible for the monitoring, backups and durability of ZooKeeper. 
+
+All code that's shared here is also available in our GitHub [repo](https://github.com/Contrast-Security-OSS/docs), so check back often for updates. You can also submit a Pull Request if you have suggestions or find any bugs in the instructions.
+
+## Before You Get Started
+Make sure you have a working installation of ZooKeeper that can be accessible from Contrast nodes. To test the connectivity from Contrast, log into that node, run a ```telnet``` command and send ```ruok```.
+
+```
+telnet localhost 2181
+Trying ::1...
+Connected to localhost.
+Escape character is '^]'.
+ruok
+imokConnection closed by foreign host.
+```
+
+In this case, ZooKeeper responds with ```imok```. You can also use other [four letter words](https://zookeeper.apache.org/doc/r3.1.2/zookeeperAdmin.html#sc_zkCommands) as ZooKeeper commands. 
+
+> **Note:** ZooKeeper is accessible through the network, anyone who has access to the ZooKeeper node can view your Contrast settings. You can address this by allowing only specified Contrast/Administrator nodes to access the ZooKeeper node.
```

Comment:
Either "Zookeeper is accessible through the network, and anyone who has..." or "Zookeeper is accessible through the network. Anyone who has...."